charts/delphai-deployment/templates/mapping.yaml

```
@@ -96,6 +96,21 @@ spec:
   {{- fail "No HTTP ports found to publish as a public API" -}}
   {{ end }} {{/* if or (eq $.Values.ports.http.enabled nil) $.Values.ports.http.enabled */}}
 
+  cors:
+    credentials: true
+    headers:
+    - Accept
+    - Authorization
+    - Content-Type
+    - grpc-timeout
+    - Origin
+    - X-GENERAL-TOKEN
+    - x-grpc-web
+    - X-Request-With
```

These we don't use at all
```suggestion
    - Accept
    - Authorization
    - Content-Type
    - Origin
```